napari/_tests/utils.py

```diff
@@ -200,7 +200,7 @@ def check_view_transform_consistency(layer, viewer, transf_dict):
         corresponding to the array of property values
     """
     if layer.multiscale:
-        return None
+        return
```

Comment:
Why is this change the opposite of all the other places where it added `return None`?

Reply:
Because this function returns only None. 

This is RET501 rule https://beta.ruff.rs/docs/rules/#flake8-return-ret